### DISCUSSION TITLE
License Key for build server

### DISCUSSION THREAD

**Discussion by 6030831** [Megabit/Blazorise#5792] [Q&A]
Hi team,  

I'm still using the Community license and considering the Professional one. 
My understanding is that we need an License Key which seems to be linked to a `user account`
I'm wondering how that work when the application is built on build server. 
Basically in my case we use GitLab CI, which run within a linux container on "root".  Also that build server is shared at the company level; build pipelines are standardize and there is very little room for customization. 

So using the account does not really make sense here; Is there another way to assign the license? 

Thanks

**Comment by stsrki** [comment]
> My understanding is that we need an License Key which seems to be linked to a `user account`

The **License Key** is linked to the developer account on https://blazorise.com/support. Once the account is created you will need to generate **Product Token**.

The Product Token should then be included in your application. See this https://blazorise.com/docs/usage/licensing/register-product-token
